career/ladder.md

```diff
@@ -195,31 +198,27 @@ Here are some examples of other career ladders:
 - [ ] Focused on and responsible for their team's productivity and growth.
 
 #### Simple
-- [ ] Has a clear understanding of long term maintenance costs and tradeoffs related to simplifying code
-- [ ] Consistently able to reduce the complexity of projects, services, and processes to get more done with less work.
-- [ ] Excels at getting the team to focus on delivering high-impact projects
+- [ ] Has solved bottlenecks in several projects throughout the years, making themselves replaceable after a planned period of time.
+- [ ] Has mentored several teams on how to reduce the complexity of projects, services, and processes to get more done with less work.
+- [ ] Excels at getting teams to focus on delivering high-impact value on the projects they mentor.
 
 #### Test-driven
-- [ ] Understands the cost of maintaining tests.
-- [ ] Can answer philosophical/economic questions like: was that test worth it for the business? Should e2e tests be run in multiple browsers if we support multiple browsers with the product? Why or why not?
-- [ ] Can facilitate dialog around reduced costs of implementation or maintenance with a team.
-- [ ] Identifies tests that are hard to maintain and fixes them proactively.
-- [ ] Frequently works with junior and intermediate engineers to reinforce a culture of testing and TDD.
-- [ ] Can support PMs in explaining the long term benefits of testing on client projects.
+- [ ] Has succesfully laid out the test strategy on legacy projects. Has changed systems from unreliable to reliable. 
+- [ ] Answers philosophical/economic questions like: was that test worth it for the business? Should e2e tests be run in multiple browsers if we support multiple browsers with the product? Why or why not?
+- [ ] Facilitates dialog around reduced costs of implementation or maintenance with a team.
+- [ ] Supports PMs in explaining the long term benefits of testing on client projects.
 - [ ] Implements security tests.
 
 #### Iterative
-- [ ] Brings new working-methods that improve team performance.
+- [ ] Has brought several new working-methods that dramatically improve the overall company performance.
 
 #### Communication
-- [ ] Works to define tasks in Asana and can mentor juniors in defining them.
-- [ ] Can collaborate with a client about estimates and prioritization.
+- [ ] Collaborates with sales about high-level estimates and prioritization.
+- [ ] Has a long track of understanding the root cause of complex situations and building a roadmap that the team can follow.
```

Comment:
What do you mean by the `building a roadmap` part.

Reply:
This focuses highly on team efficiency, rather than on the developer itself. When a project is complex, we want a principal engineer to be able to understand what it would take to change it and plan with the PM for the necessary change. The principal engineer will not necessarily be the developer who will end up doing the changes. Instead, other less-experienced developers will be able to solve the problem.

Reply:
So something like `Has a long track record of suggesting ways to simplify complex apps, that the team can then implement`?

Reply:
Good suggestion. I'll change it.